### PR TITLE
Create hardware.md

### PR DESCRIPTION
**Pull request by knmcguire**

### CODE REVIEW

File: hardware.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+### Companion Computers
+For the drones that can carry it, the companion computers are important since they can do additional computation that the flight controller can not easily do.
+As these are capable of running some form of Linux, these can handle for instance  computer vision with [OpenCV](https://opencv.org/) or run nodes with [ROS](https://www.ros.org/)
+
+- [Nvidia Jetson Xavier](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-xavier-nx/) or the [TX2 Module](https://developer.nvidia.com/embedded/jetson-tx2)
+- [Intel Nuc Boards](https://www.intel.com/content/www/us/en/products/details/nuc/boards/products.html)
+- [Raspberry PI 3 (A+)](https://www.raspberrypi.com/products/raspberry-pi-3-model-a-plus/) or [Raspberry Pi 4](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/)
+- [Khadas Vim3](https://www.khadas.com/vim3)
+- [Odroid (various boards](https://www.hardkernel.com/product-category/odroid-board/)
```
Comment (mrpollo) [inline hardware.md:52]:
```suggestion
- [Odroid (various boards](https://www.hardkernel.com/product-category/odroid-board/)
- [NXP NAVQPlus](https://www.nxp.com/design/designs/navqplus-ai-ml-companion-computer-evk-for-mobile-robotics-ros-ground-stations-and-camera-heads:8MPNAVQ)
- [Qualcomm RB5](https://developer.qualcomm.com/qualcomm-robotics-rb5-kit)
```

**Comment by mrpollo** [comment]
Hey @knmcguire I think we can merge if we add a WIP notice at the top

File: hardware.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Aerial robotics Hardware
```
Comment (mrpollo) [inline hardware.md:1]:
```suggestion
# Work In Progress
We are still hard at work at mapping the hardware landscape, if you see anything missing please send a PR, we want to encourage manufacturers to help us make this a complete list

# Aerial robotics Hardware
```

**Review by mrpollo — COMMENTED** [review]
Hey @knmcguire I added a few changes

File: hardware.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Many of the UAVs are usually built by hand and composed of different components. This usually consists of a drone frame, fligth controller boards, companion computers and of course motors, batteries and ESCs. 
+
+### Drone Frames
+Many drone frames are usually built from carbon fiber and custom-made for application or research.
+There are some frames that are provided that will provide some base:  
+- [DJI Flame wheel ARF kit F550, F450, F330](https://www-v1.dji.com/flame-wheel-arf/feature.html)
+
+### Flight controllers
+- [Pixhawk 4](https://docs.px4.io/main/en/flight_controller/pixhawk4.html) discontinued, go look at:
+   -  [Holybro's Pixhawk 6C](https://holybro.com/collections/autopilot-flight-controllers/products/pixhawk-6c)
```
[the remark this answers is not in the document]
Comment (knmcguire) [reply hardware.md:43]: Ah yes absolutely. Let's do that.

**Comment by knmcguire** [comment]
Thanks @mrpollo! I've added your changes and going to merge it

**Pull request merged**